### DISCUSSION TITLE
Github pages showing 404 error.

### DISCUSSION THREAD

**Discussion by janardhan345** [community/community#146604] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

i have tried everything told in the troubleshooting error web. but still its not working 
![image](https://github.com/user-attachments/assets/7f298904-9c62-489e-9019-f764473787db)

**Comment by mecodeatlas** [comment, marked as answer]
Hey @janardhan345!

Did you get a chance to check out our [Troubleshooting forum](https://github.com/orgs/community/discussions/23525) for 404 errors and review our [documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/troubleshooting-404-errors-for-github-pages-sites)? If you've already followed the suggested steps and guidelines, we recommend taking a look at [this discussion](https://github.com/orgs/community/discussions/23525), as it might be helpful in resolving the issue :)
